### DISCUSSION TITLE
Access to wikis in private repository

### DISCUSSION THREAD

**Discussion by nonsonopippo** [community/community#73606] [New to GitHub] [answered]
Hello!

I am planning to upgrade to a Pro account in order to create a wiki in a private repository.
My question is: are collaborators, allowed to enter the repository, also allowed to modify/create/delete files in the wiki, even if they have a free plan? Can I be the only one with a pro plan, if I am the "head" of the private wiki, with all the collaborators contributing while on free plans?

Thanks a lot,
Alessandro

**Comment by Naman-B-Parlecha** [comment, marked as answer]
Yes, in GitHub, if you have a Pro or higher plan and you're the owner or have admin access to a repository, you can add contributors who are on the free version to your wiki. The access permissions for the wiki are typically tied to the repository's access settings, so as long as you have the necessary access to the repository, you can manage contributors to the wiki, regardless of their GitHub plan

**Reply by Naman-B-Parlecha** [reply]
If u find this helpful please mark this as the answer and if any more questions just tag me thanks 👍

**Reply by nonsonopippo** [reply]
That is very helpful, thanks a lot!

**Reply by Naman-B-Parlecha** [reply]
Good luck with your wiki bro

**Comment by webmemo-code** [comment]
Does this work? I am on a Pro license but when I want to activate a Wiki, I am told to upgrade. I am not part of an organization.
